app/airq/models/events.py

```diff
@@ -105,6 +107,10 @@ def _get_schema(self) -> EventSchema:
             return UnsubscribeEventSchema
         elif self.type_code == EventType.ALERT:
             return AlertEventSchema
+        elif self.type_code == EventType.FEEDBACK_BEGIN:
+            return FeedbackEventSchema
+        elif self.type_code == EventType.FEEDBACK_RECIEVED:
+            return FeedbackEventSchema
```

Comment:
Maybe for this one we should store the actual feedback?

Reply:
+1